frontend/app/src/filters.ts

```diff
@@ -20,12 +20,14 @@ export function truncateAddress(address: string, truncLength = 4): string {
     ? 4
     : 0;
 
-  if (address.length <= truncLength * 2 + startPadding) {
+  const length = address.length;
+
+  if (length <= truncLength * 2 + startPadding) {
     return address;
   }
 
-  return `${address.slice(0, truncLength + startPadding)}...${address!.slice(
-    address!.length - truncLength,
-    address!.length
+  return `${address.slice(0, truncLength + startPadding)}...${address.slice(
+    length - truncLength,
+    length
```

Comment:
After the release, we must remove the `filters.t`s and move the utilities under `src/utils/.

This is a leftover from before we started the migration.

Reply:
I think we can also do that after this PR, just moving the file right